Docs/source/Software/BSP.rst

```diff
@@ -135,3 +135,23 @@ Usage
 Additional Considerations
     Increasing the clock speed does take more power. Although this is minor considering how much the
     entire BPS takes, it is worth noting.
+
+SPI BSP: Clark Poon, Sijin Woo, and Sugam Arora
+===============================================
+
+Purpose
+    SPI is used for communication between the uC and the minion LTC6811s in the system.
+
+Usage
+    Our car uses the ``SPI1`` and ``SPI3`` busses. The ``SPI1`` bus is for communication with the LTC6811s
+    and the ``SPI3`` bus is for communication with the AS8510 current sensor. When calling the init function, 
+    you must specify which port you intend to use (via the ``spi_port_t`` struct). The user can read/write on whichever bus they 
+    have chosen. 
+    This BSP includes IRQ Handlers for both buses that post the ``MinionsASIC_Mutex``.
+
+    ``SPI_Wait()`` has two different versions - one for the bare-metal BPS code and one for the RTOS code. 
+    To use one of the versions, the user must have a line containing ``#define BAREMETAL`` or ``#define RTOS``
+    in their code.
+
+Additional Considerations
+    The ``SPI1`` and ``SPI3`` IRQs are only included in the RTOS version of the code.
```

Comment:
Also include that if additional SPI buses are needed for other drivers, they just have to edit the spi_port_t struct typed variable.

Reply:
More information is probably included in Chase's comments in the file.